frontend/play/index.html

```diff
@@ -184,6 +184,7 @@ <h1>Share</h1>
           <button class="icon-close"></button>
         </header>
         <main>
+          <p class="note">⚠️ Code has <span class="err">parse errors</span>!</p>
```

Comment:
Is "parse error" the best term here? It's technically correct, but a bit jargony.

Reply:
i'll change it to just `Code has errors!` ok?